nucypher/utilities/prometheus/collector.py

```diff
@@ -121,6 +124,7 @@ def _collect_internal(self) -> None:
 
         self.metrics["learning_status"].state('running' if self.ursula._learning_task.running else 'stopped')
         self.metrics["known_nodes_gauge"].set(len(self.ursula.known_nodes))
+        self.metrics["availability_score_gauge"].set(len(self.ursula._availability_tracker.score))
```

Comment:
I'm pretty sure this is the correct reference according to https://github.com/nucypher/nucypher/blob/main/tests/acceptance/network/test_availability.py#L41:

```
assert ursula._availability_tracker.score == 10
```

Hoping a reviewer can verify that this will export the correct availability score.